config/test.exs

```diff
@@ -2,3 +2,5 @@ use Mix.Config
 
 config :elix, Elix.Robot,
   adapter: Hedwig.Adapters.Test
+
+config :elix, :giphy_client, Elix.GiphyClientMock
```

Comment:
Hm, I get a compile warning when running `mix test` that `Elix.GiphyClientMock` is not available:

```elixir
warning: function Elix.GiphyClientMock.get/1 is undefined (module Elix.GiphyClientMock is not available)
  lib/elix/responders/gif_me.ex:41
```

The tests still pass because I manually `require` the file in `test_helper.exs`, but I'd like to avoid the compiler warning by making this file available at compile time. (Changing the extension to `.exs` doesn't avoid the compile warning. I should figure out how to do that.